data/nmstate/operand/operator.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
     app: kubernetes-nmstate
     component: kubernetes-nmstate-webhook
 spec:
-  replicas: 2
+  replicas: {{ .WebhookReplicas }}
```

Comment:
This file is generated through `make bump-nmstate`, I believe this change would be overwritten by that.

We fetch manifest templates from https://github.com/nmstate/kubernetes-nmstate/blob/main/deploy/handler/operator.yaml and replace variables set there with ours when needed (when the original name is not already fine).

Would you please contribute a PR to kubernetes-nmstate extending the template there? It does not need to be dynamic there (that will come in a follow-up PR) - adding the variable there and setting it always to 2 would be good enough.

Reply:
ahh 
Thank you 
I will open a PR there.

Reply:
[Done.](https://github.com/nmstate/kubernetes-nmstate/pull/810)